src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -371,6 +376,28 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
             {
                 emitShift(intrin.op2, op1Reg);
             }
+
+        }
+        else if (intrin.category == HW_Category_EnumPattern)
+        {
+            assert(hasImmediateOperand);
+
+            switch (intrin.numOperands)
+            {
+                case 1:
+                    {
+                        HWIntrinsicImmOpHelper helper(this, intrin.op1, node);
+                        for (helper.EmitBegin(); !helper.Done(); helper.EmitCaseEnd())
+                        {
+                            const insSvePattern pattern = (insSvePattern) helper.ImmValue();
+                            GetEmitter()->emitIns_R_PATTERN(ins, emitSize, targetReg, opt, pattern);
```

Comment:
how are you getting `ins == INS_sve_ptrue` here? You have set the instruction to `INS_invalid` in `hwinstrinsiclistarm64sve.h`?

Reply:
yes, I didn't notice it earlier.